GTMAppAuth/Sources/KeychainStore/KeychainStore.swift

```diff
@@ -110,7 +110,7 @@ extension KeychainStore: AuthSessionStore {
   }
 
   @objc(saveAuthSession:forItemName:error:)
```

Comment:
As you pointed out, we need to update the `@objc` selector.

Reply:
Forgot to push up the change.